content/docs/tech-resources/deep-links-self-serve.md

```diff
@@ -1,5 +1,5 @@
 ---
-lastUpdated: "02/24/2021"
+lastUpdated: "12/23/2021"
```

Comment:
Whoops! I missed this one. @bkemper pointed out in a previous file change that we shouldn't update the date if we're not updating content. Can we revert this date back to "02/24/2021"?

Reply:
Done. I suppose the link updates don't really change the content.

Reply:
Yeah, not if they're fixing a problem and going to the same basic content. Thanks, @tuck1s !